backend/promis/backend_api/urls.py

```diff
@@ -28,8 +29,10 @@
 '''
 
 urlpatterns =  router.urls + userreg.urls + [
-    url('^api-auth/', include('rest_framework.urls', namespace = 'rest_framework'))
-    ] + docrout.urls
+    url('^api-auth/', include('rest_framework.urls', namespace = 'rest_framework')),
+    url(r'profile/$', views.UserProfile.as_view(), name = 'profile')
```

Comment:
Couple of questions here:
1. Does that mean we have 2 distinct urls for registering and querying the profile? 
2. Can you list all the URLs involved in this so that I can make nginx rules for them?